### ISSUE TITLE
After pod install with AFNetworking 4.0.x throws exception

### ISSUE THREAD

**Issue by nikhilpatel7** [deanWombourne/AFNetworking-streaming#5]
Build Error: No visible @interface for 'AFHTTPSessionManager' declares the selector 'GET:parameters:success:failure:' 

Above build error occurs while building with AFNetworking version 4.0.x. I had to upgrade to 4.0.x due to WKWebView mandatory upgrade.

```
#pragma clang diagnostic push
#pragma clang diagnostic ignored "-Wdeprecated-declarations"
        task = [super GET:URLString
               parameters:parameters
                  success:^(NSURLSessionDataTask *task, __unused id response) {
                      if (success)
                          success(task);
                  }
                  failure:failure];
#pragma clang diagnostic pop
```
After spending some time behind this I also fixed this issue locally with following changes:

```
#pragma clang diagnostic push
#pragma clang diagnostic ignored "-Wdeprecated-declarations"
        task = [super GET:URLString
               parameters:parameters
                headers:nil
                 progress:nil
                  success:^(NSURLSessionDataTask *task, __unused id response) {
                      if (success)
                          success(task);
                  }
                  failure:failure];
#pragma clang diagnostic pop
```

**Comment by nikhilpatel7** [comment]
Can anyone fix this is library itself so I don't need to make this change manually everytime. I don't problem doing it but when it comes to automated build it becomes painful.
Thanks

**Comment by deanWombourne** [comment]
Hi,

Can you check the latest commit on master to see if I've fixed it please? The trick is fixing it for AFNetworking 4.x without breaking if for all the other versions :)

i.e. something like this in your podfile:

```
 pod 'AFNetworking-streaming', :git => 'https://github.com/deanWombourne/AFNetworking-streaming.git', :branch => 'master'
```

**Comment by deanWombourne** [comment]
Sigh, that's completely my fault! The line should have been this (spot the difference!):

```
pod 'AFNetworking+streaming', :git => 'https://github.com/deanWombourne/AFNetworking-streaming.git', :branch => 'master'
```

**Comment by nikhilpatel7** [comment]
Ahh. Even I didn't realise that small thing... It worked smoothly. Thanks for your time. :)

**Comment by deanWombourne** [comment]
Awesome. I've just released these changes as version `1.1.1` so you don't have to point directly at master anymore :)

**Comment by deanWombourne** [comment]
Let me know if that's working, and I'll close this ticket.

Thanks,
Sam
